CONTRIBUTING.md

```diff
@@ -66,6 +66,111 @@ For first-time contributers, we suggest to look for issues categorized as [trivi
 
 If you want a hassle-free contribution look for issues categorized as [preapproved](https://issues.dlang.org/buglist.cgi?component=dmd&keywords=preapproved&product=D).
 
+## DMD Style
```

Comment:
This text in this section seems to be less about style and more about best practices and guidelines.